extensions/amp-consent/amp-consent.md

```diff
@@ -118,6 +126,8 @@ AMP expects the response to be a JSON object like the following:
 }
 ```
 
+For granular consent, this response can also contain a map of `purposeConsent` choices.
```

Comment:
An example would be nice here.

Reply:
can you help with that? I didn't include one because I wasn't 100% sure how that would look...

Reply:
I want to merge this so that it can get out there right away, but I'd propose this be done in a followup PR!